cmd/awscollector/Dockerfile

```diff
@@ -69,11 +70,13 @@ FROM scratch
 
 COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
 COPY --from=package /workspace/awscollector /awscollector
+COPY --from=package /workspace/healthcheck /healthcheck
 COPY --from=package /workspace/config/ /etc/
 
 ENV RUN_IN_CONTAINER="True"
 # aws-sdk-go needs $HOME to look up shared credentials
 ENV HOME=/root
 ENTRYPOINT ["/awscollector"]
 CMD ["--config=/etc/otel-config.yaml"]
+HEALTHCHECK --interval=1s --timeout=1s --start-period=2s --retries=3 CMD [ "/healthcheck" ]
```

Comment:
Do we want to add this to the docker file explicitly or allow a customer to choose to use this? Can this be turned off? Would we want it to be turned off?

Reply:
Thanks. Done